### DISCUSSION TITLE
Feedback: https://nitro.unjs.io/deploy/workers is 404 on direct request

### DISCUSSION THREAD

**Discussion by janpio** [nitrojs/nitro#1404] [General]
I found a link to https://nitro.unjs.io/deploy/workers via Google, but accessing it via there or directly is a 404.
If I click the logo to get to the Nitro docs homepage, and then use the back button I suddenly see the content.
I don't think this is expected.

**Comment by Hebilicious** [comment]
Hi @janpio, thanks for reporting ! We have an open issue here https://github.com/unjs/nitro/issues/1394

**Reply by janpio** [reply]
Ah cool. Seems I should have used the search with the URL... oops. Sorry.
